app/src/main/report/components/Versions/VersionDetails.tsx

```diff
@@ -8,34 +8,41 @@ import {ParameterList} from "../Parameters/ParameterList";
 import {DataLinks} from "../Data/DataLinks";
 import {ResourceLinks} from "../Resources/ResourceLinks";
 import {ArtefactsList} from "../Artefacts/ArtefactsList";
-import { FileDownloadLink } from "../FileDownloadLink";
+import {FileDownloadLink} from "../FileDownloadLink";
+import {ReportVersionSwitcher} from "./ReportVersionSwitcher";
+import {IRouter} from "simple-react-router";
 
 const styles = require("../../styles/reports.css");
 
-export interface VersionProps extends RemoteContent {
-    versionDetails: Version,
-    report: string
+interface PublicProps {
+    onChangeVersion: (version: string) => void;
+}
+
+export interface VersionProps extends RemoteContent, PublicProps {
+    versionDetails: Version;
+    report: string;
+    otherVersions: string[];
```

Comment:
I think this shoudl be `allVersions` rather than `otherVersions` right?

Reply:
Fixed